test/e2e/plain_provisioner_test.go

```diff
@@ -1294,7 +1295,7 @@ var _ = Describe("plain provisioner garbage collection", func() {
 					return false
 				}
 				installedBundleName := bi.Status.InstalledBundleName
-				return installedBundleName != "" && installedBundleName != originalBundleName
+				return installedBundleName != ""
```

Comment:
This check still seems relevant given we're testing that a new Bundle has been created?

Reply:
Yes, it's definitely relevant, but the name is now the same between the original bundle and the new bundle, since the template hasn't changed in this test, we just delete/recreate the Bundle. I put these e2e changes in their own commit to highlight. 

Maybe comparing something else here, like resourceVersion, would help indicate that the resources are different even though the name is the same?

Reply:
Actually, maybe the UID makes more sense in this context